### ISSUE TITLE
URL support in card description

### ISSUE THREAD

**Issue by lordofscripts** [kanriapp/kanri#718]
**Is your feature request related to a problem? Please describe.**
Not related to a problem.

**Describe the solution you'd like**
That the user can use hyperlinks in the card's text description in a manner like HTML documents where the URL is only visible when editing the hyperlink, but that in normal render mode only the URL text is rendered with underline decoration..

**Describe alternatives you've considered**

**Additional context**

**Comment by trobonox** [comment]
I've tried to add this before but it is currently prevented by a security feature in the app framework Tauri, which doesn't allow opening all links entered in an app for security purposes. I'm not sure if I want to disable this because of said security concerns, but I might consider it once I migrate to a higher version of Tauri which includes a lot of new and updated features.

**Comment by lordofscripts** [comment]
Good point but, who would put a malicious URL in their own Kanban? Doesn't look like an issue for personal, single-user Kanbans. Now, if the Kanban board is shared by multiple people it COULD be an issue. Then again, I wouldn't add people I can't trust in my Kanban. In any case, I only use it for myself (Personal projects, Creative writing & Software development pet projects).
  Anyway, I am super happy that there is now a build for my platform (as of 0.8). 👍

**Comment by badlydrawnrob** [comment]
@lordofscripts @trobonox I also suggested this feature in #619, which I think is a reasonable one, but I don't know what security implications it'll have if you've gotta strip any malicious code out. Perhaps in the interim you could just colour them (perhaps `[` and `](link.com)` in grey so they're distinct from the rest of the text)? I dunno. I'm using https://tinyurl.com/ to make them less ugly. Kind of surprised this is a security risk if Markdown is supported in other cases.

**Comment by trobonox** [comment]
I can imagine one case where this could be misused, but that would require an attacker to already have full access to your filesystem and then you very likely already have other problems than a potentially malicious link being injected into one of your Kanri boards.
I am strongly considering this feature since I definitely see the use for it, but at the moment development is very slow in general so no guarantee or any estimates I can give at the moment.

**Comment by lordofscripts** [comment]
Well, if security is a concern for a board that isn't shared (they are all single-user), then it might be configured as an application option in Settings (Allow hyperlinks).
